resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -299,6 +300,20 @@ private[spark] object Config extends Logging {
     .stringConf
     .createOptional
 
+  val KUBERNETES_JOB_MIN_CPU = ConfigBuilder("spark.kubernetes.job.minCPU")
```

Comment:
BTW, this naming looks inconsistent to me. For the other K8s configuration, we use `Core` instead of `CPU`. Please double check this.

Reply:
Yes, this might be a good suggestion for improvement. I also searched in spark, only `spark.task.cpus` are using `cpu`, and others are using `cores`. I guess it might be because of the name originally inherited from Yarn.

So, I think `cores` is also fine for me.

Reply:
also cc @holdenk